README.md

```diff
@@ -1,11 +1,54 @@
-# vanilla-monorepo
-Proof-of-concept / exploration of the Vanilla design system as a monorepo
+# DS-25 - Canonical's new design system
 
-## Installation
+Canonical's new design system.
+This is the successor to [Vanilla framework](https://vanillaframework.io).
 
-```sh   
-bun i
-git config diff.lockb.textconv bun
-git config diff.lockb.binary true
-```
+## Getting started
 
+1. Install [NodeJS](https://nodejs.org/en/download/package-manager) version 20 or later.
```

Comment:
Do you need node, if we use bun?

Reply:
Technically no, but it is necessary for testing the boilerplate in #19 . I ran into an error running local package bin scripts in bun workspaces with `bunx` that I have been circumventing by using `npx`. 

Filed an [issue](https://github.com/oven-sh/bun/issues/15036) with Bun. 
There is a similar [issue](https://github.com/oven-sh/bun/issues/11729) for running package bin scripts with local filepaths, so there currently isn't a way for bun to run a package bin script unless it is published to a package registry first.